### PR TITLE
OPS: Manual upload to Testflight

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/e45htsgezjq2enzgmwxfunzwem?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12141)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/mjx55ecnii6anxzv57aifxopnu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12146)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/674x4ovvvjoqldkhafgjhnnviu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12151)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xmy3vyqy6e7qi7dqbjvksg6oq4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12153)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kclhorgoanangwcf2btykqwamm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12154)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/bn4unu53xl7vupm6tfxhb5lg4m?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/12158)

File: .github/workflows/build-ios-release-pullrequest.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        run: sudo gem install cocoapods
+
+      - name: Clear Derived Data
+        run: bundle exec fastlane ios clear_derived_data_lane
+        working-directory: ./ios
+
+      - name: Install node_modules
+        run: npm install
+
+      - name: Display release-notes.txt
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-pullrequest.yml:61]: duplicate


```
      - name: Display release-notes.txt
        run: cat release-notes.txt

      - name: Display release-notes.json
        run: cat release-notes.json
```

File: .github/workflows/build-ios-release-pullrequest.yml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        run: cat release-notes.txt
+
+      - name: Display release-notes.json
+        run: cat release-notes.json
+
+      - name: Set up Git Authentication
+        env:
+          ACCESS_TOKEN: ${{ secrets.GIT_ACCESS_TOKEN }}
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-pullrequest.yml:69]: remind me where `GIT_ACCESS_TOKEN` comes from? you created the token and put it in env vars?

File: .github/workflows/build-ios-release-pullrequest.yml [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+
+      - name: Generate Build Number based on timestamp
+        run: echo "NEW_BUILD_NUMBER=$(date +%s)" >> $GITHUB_ENV
+
+      - name: Set Build Number
+        run: bundle exec fastlane ios increment_build_number_lane
+        working-directory: ./ios
+        env:
+          NEW_BUILD_NUMBER: ${{ env.NEW_BUILD_NUMBER }}
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-pullrequest.yml:98]: this makes no sense to me. provide env var from existing env var with the same name..?

Comment (Overtorment) [reply .github/workflows/build-ios-release-pullrequest.yml:98]: looked at docs.  no need to explicitly pass it, it should just be available  after youve set it in `$GITHUB_ENV` in prev step

File: .github/workflows/build-ios-release-pullrequest.yml [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+        run: bundle exec fastlane ios build_app_lane
+        working-directory: ./ios
+        env:
+          NEW_BUILD_NUMBER: ${{ env.NEW_BUILD_NUMBER }}
+      
+      - name: Get Latest Commit Message
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.upload_to_testflight == 'true'
+        run: |
+          LATEST_COMMIT_MESSAGE=$(git log -1 --pretty=format:"%s")
+          echo "LATEST_COMMIT_MESSAGE=${LATEST_COMMIT_MESSAGE}" >> $GITHUB_ENV
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-pullrequest.yml:114]: i think putting branch name in testflight release notes might be a better idea

Comment (Overtorment) [reply .github/workflows/build-ios-release-pullrequest.yml:114]: btw i dont see where `LATEST_COMMIT_MESSAGE` is used later

File: .github/workflows/build-ios-release-pullrequest.yml [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+      - name: Create App Store Connect API Key JSON
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.upload_to_testflight == 'true'
+        run: echo '${{ secrets.APP_STORE_CONNECT_API_KEY_CONTENT }}' > ./ios/appstore_api_key.json
+
+      - name: Upload to TestFlight
+        if: github.event_name == 'workflow_dispatch' && github.event.inputs.upload_to_testflight == 'true'
+        run: echo "Uploading to TestFlight..."
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-pullrequest.yml:122]: wheres the actual upload..?

**Review by Overtorment — CHANGES_REQUESTED** [review]

**Pull request closed without merging**